README.md

```diff
@@ -1,8 +1,62 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) Completed
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Payal Gupta . Jarvis Consulting
 
+I am a Results-driven individual who is passionate about software development. I have more than 2 years of experience in the insurance/financial industry as a Quality Assurance Analyst, this experience helped me to build an analytical, detail-oriented mindset and also exposed me to the work culture and methodologies followed in large-scale industry. While I was completing my Bachelor in Computer Science Engineering I could help a non-profit organization by developing their website which helped them to raise funds, this made me more passionate about this field. Since then I have worked on front-end projects using JavaScript, HTML/CSS. Due to my curious personality, I kept on learning the latest technologies like Cloud services(AWS and GCP) side by side. While working in Jarvis I worked on the industry-related application using technologies like Java, Maven, Docker, SQL, Bash, REST API. Through my passion to help others by improving existing methods and enthusiasm to learn new emerging technologies, I will be a great asset to the team and will also be excited about the new upcoming opportunities.
+
+## Skills
+
+**Proficient:** Java, Python, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** Docker, HTML/CSS, Maven, Selenium, BigQuery(Google Cloud Platform), Dataproc(Google Cloud Platform)
+
+**Familiar:** EC2 (Amazon Web Services), Pandas, Numpy, DynamoDB, Hadoop
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_PayalGupta](https://github.com/jarviscanada/jarvis_data_PayalGupta)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_PayalGupta/tree/master/linux_sql)]: Implemented a Linux clustering monitoring agent to monitor, collect and analyze data from host machines in a 10 node cluster over a network in a PostgreSQL database provisioned using docker container using bash scripts for better business decisions and resources planning.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_PayalGupta/tree/master/core_java)]:
+      
+  - Twitter App: Developed a CLI interface using MVC architecture for users to create, read and delete tweets through Twitter RESTful API. Performed testing using JUnit test cases.
```

Comment:
missing keywords: Java 8, docker, Maven, Intellij, Postman, Mockito. Follow the SDLC pattern (design, implementation, test, deployment (maven and docker).